src/julia/Integer.jl

```diff
@@ -366,14 +366,14 @@ function root(a::T, n::Int; check::Bool=true) where T <: Integer
    end
 end
 
-moduli3 = [7, 8, 13]
-residues3 = [[0, 1, 6], [0, 1, 3, 5, 7], [0, 1, 5, 8, 12]]
+const moduli3 = [7, 8, 13]
+const residues3 = [[0, 1, 6], [0, 1, 3, 5, 7], [0, 1, 5, 8, 12]]
 
-moduli5 = [8, 11, 31]
-residues5 = [[0, 1, 3, 5, 7], [0, 1, 10], [0, 1, 5, 6, 25, 26, 30]]
+const moduli5 = [8, 11, 31]
+const residues5 = [[0, 1, 3, 5, 7], [0, 1, 10], [0, 1, 5, 6, 25, 26, 30]]
 
-moduli7 = [8, 29, 43]
-residues7 = [[0, 1, 3, 5, 7], [0, 1, 12, 17, 28], [0, 1, 6, 7, 36, 37, 42]]
+const moduli7 = [8, 29, 43]
+const residues7 = [[0, 1, 3, 5, 7], [0, 1, 12, 17, 28], [0, 1, 6, 7, 36, 37, 42]]
```

Comment:
I was curious what these vectors are used for. Turns put the code is buggy: the code in line 391 below checks for divisibility by 3 instead of 7... oops.

Reply:
I am also confused by the mix of % and mod. Can you provide a PR fixing this function or make an issue so we don't forget?

Reply:
I only have my phone right now